the older python on the host expects LIBDIRNAME here.
once it gets upgraded to 2.6.8, we can drop this patch.

--- Makefile.pre.in
+++ Makefile.pre.in
@@ -87,6 +87,7 @@
 INCLUDEDIR=	@includedir@
 CONFINCLUDEDIR=	$(exec_prefix)/include
 SCRIPTDIR=	$(prefix)/@@GENTOO_LIBDIR@@
+LIBDIRNAME=	@@GENTOO_LIBDIR@@
 
 # Detailed destination directories
 BINLIBDEST=	$(LIBDIR)/python$(VERSION)
